api/src/main/java/io/opentelemetry/api/metrics/Meter.java

```diff
@@ -25,6 +25,14 @@
 @ThreadSafe
 public interface Meter {
 
+  /**
+   * Returns a {@link Meter} that only creates no-op {@link Instrument}s that neither record nor are
+   * emitted.
+   */
+  static Meter noop() {
+    return DefaultMeter.getInstance();
```

Comment:
The name of the method is misleading, for metrics it is right now a noop but we may add small functionality. Not sure about a better name but wanted to raise this issue

Reply:
Hmm - how about just `getDefault` then? An unfortunately generic name but it does give us flexibility to change so it seems like a better name.

Reply:
yeah, I think `getDefault` might be better, even though I much prefer saying "NOOP" loudly whenever I read the code. ;)